doc/whats_new.rst

```diff
@@ -12,6 +12,8 @@ Changelog
 
     - Add `overlay_times` parameter to :func:`mne.viz.plot_epochs_image` to be able to display for example reaction times on top of the images, by `Alex Gramfort`_
 
+    - Add :class:`mne.Annotations` for for annotating segments of raw data by `Jaakko Leppakangas`_
```

Comment:
ahh, right, I forgot we just do it as `raw.annotations = `. Never mind, it should be fine.